crates/viewer/re_time_panel/tests/time_panel_tests.rs

```diff
@@ -0,0 +1,66 @@
+use egui::{CentralPanel, Vec2};
+use re_chunk_store::{Chunk, LatestAtQuery, RowId};
+use re_log_types::example_components::MyPoint;
+use re_log_types::external::re_types_core::Component;
+use re_log_types::{build_frame_nr, EntityPath};
+use re_time_panel::TimePanel;
+use re_viewer_context::blueprint_timeline;
+use re_viewer_context::test_context::TestContext;
+use re_viewport_blueprint::ViewportBlueprint;
+use std::sync::Arc;
+
+#[test]
+pub fn time_panel_should_match_snapshot() {
+    let mut test_context = TestContext::default();
+    let mut panel = TimePanel::default();
+
+    let points1 = MyPoint::from_iter(0..1);
+
+    for i in 0..2 {
+        let entity_path = EntityPath::from(format!("/entity/{i}"));
+        let mut builder = Chunk::builder(entity_path.clone());
+        for frame in [10, 11, 12, 15, 18, 100, 102, 104].map(|frame| frame + i) {
```

Comment:
I think we should add a section with some actually density so we have a test for this look:

![image](https://github.com/user-attachments/assets/d019d5aa-9d71-4849-93c3-7078b2d6246f)